tools/devctr/Dockerfile.x86_64

```diff
@@ -16,7 +16,7 @@ ENV CARGO_HOME=/usr/local/rust
 ENV RUSTUP_HOME=/usr/local/rust
 ENV PATH="$PATH:$CARGO_HOME/bin"
 
-# Install system dependecies
+# Install system dependencies
 #
 RUN apt-get update \
```

Comment:
You can use `cargo install cargo-audit --version 0.10.0` to keep the current version of audit and solve the 1.43 incompatibility as a future step. Do not forget to remove also `musl-tools` from the docker file.